.github/workflows/pr-comment.yml

```diff
@@ -40,13 +40,14 @@ jobs:
             const totalCommits = result.repository.pullRequest.commits.totalCount;
             // Get unique email addresses
             const commitEmails = [... new Set(result.repository.pullRequest.commits.nodes.map((node) => node.commit.author.email))];
+            // Check that all emails end in "@ccpo.mil"
             const allEmailCcpo = commitEmails.every((email) => email.endsWith("@ccpo.mil"));
 
             core.setOutput("commit-count", totalCommits);
             core.setOutput("all-email-ccpo", allEmailCcpo);
 
       - name: Message on too many commits
-        if: steps.get-commit-data.outputs.commit-count > 250
+        if: ${{ fromJSON(steps.get-commit-data.outputs.commit-count) > 250 }}
```

Comment:
I think you said it earlier, but where does this limit come from?  Beyond this limit we need ensure we perform the check manually?